benchmarks/src/main/scala/zio/zmx/package.scala

```diff
@@ -0,0 +1,30 @@
+package zio
+
+package object zmx {
+  def spawn(i: Int): ZIO[Any, Nothing, Fiber.Runtime[Nothing, Unit]] =
+    if (i <= 0) {
+      ZIO.never.fork
+    } else
+      for {
+        rec <- spawn(i - 1).fork
+        f   <- rec.join
+      } yield f
+
+  def deep(size: Int) =
+    for {
+      _ <- spawn(size)
+    } yield ()
+
+  def broad(size: Int) =
+    for {
+      _ <- ZIO.foreach(1 to size)(_ => ZIO.never.fork)
```

Comment:
```suggestion
      _ <- ZIO.foreachPar(1 to size)(_ => ZIO.unit)
```

Reply:
updated in https://github.com/zio/zio-zmx/pull/135/commits/cab61eb7f96d55c11127a95f3dc32f732ffa020f